distributed/distributed.yaml

```diff
@@ -71,6 +71,15 @@ distributed:
     socket-backlog: 2048
     recent-messages-log-length: 0  # number of messages to keep for debugging
 
+    # Facebook's Zstandard - Fast real-time compression algorithm;  reference https://pypi.org/project/zstandard/
+    zstd:
+      import_policy: default    # one of "default", "cffi_fallback", "cext", "cffi"; use as zstandard module import policy.
```

Comment:
I agree with @mrocklin here. While the thread count and compression level configs make totally sense, I would also not monkey-patch the OS env variables, especially since the user has other ways of doing so. The proposed approach might not even work, since we are now relying on the fact the `distributed` is the first importer, which might be wrong in so many cases.

Reply:
@crepererum   good point. :) 
@crepererum  @mrocklin
    now we use `importlib.reload(zstd)` to do reload the zstandard module to provide user unified design and consistent interaction.


To github.com:abaelhe/distributed.git
   bca1833c..775b2975  patch-5 -> patch-5